browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -1042,6 +1042,120 @@ test.describe('applicant program index page', () => {
           })
         })
       })
+
+      test.describe('common intake form', () => {
+        const commonIntakeFormProgramName = 'Benefits finder'
+
+        test.beforeEach(async ({page, adminPrograms}) => {
+          await loginAsAdmin(page)
+
+          await adminPrograms.addProgram(
+            commonIntakeFormProgramName,
+            'program description',
+            'short program description',
+            'https://usa.gov',
+            ProgramVisibility.PUBLIC,
+            'admin description',
+            /* isCommonIntake= */ true,
+          )
+
+          await adminPrograms.addProgramBlockUsingSpec(
+            commonIntakeFormProgramName,
+            {
+              name: 'Screen 2',
+              description: 'first block',
+              questions: [{name: 'first-q'}],
+            },
+          )
+          await adminPrograms.publishAllDrafts()
+          await logout(page)
+        })
+
+        test('shows common intake form card when an application has not been started', async ({
+          page,
+          applicantQuestions,
+        }) => {
+          await validateScreenshot(
+            page.getByLabel('Get Started'),
+            'ns-common-intake-form',
+          )
+          await applicantQuestions.expectProgramsWithFilteringEnabled(
+            {
+              expectedProgramsInMyApplicationsSection: [],
+              expectedProgramsInProgramsAndServicesSection: [
```

Comment:
question (non-blocking): This isn't really testing that the common intake form shows up where expected, but it's testing it doesn't show up in the programs and services section. What do you think about adding another param to this helper method that's for expected programs in the common intake section?

Reply:
Although the `page.getByLabel('Get Started')` above would error if there's nothing there, so maybe that's good enough.